nodes/main.tf

```diff
@@ -69,6 +69,7 @@ module "workers" {
   max_nodes             = "${var.max_nodes_per_az}"
   node_group_name       = "${var.node_group_name}"
   ami_image_id          = "${var.ami_image_id}"
+  default_worker_instance_type = "${var.default_worker_instance_type}"
```

Comment:
Can you please run `terraform fmt` on this file? should fix the alignment

Reply:
Done